docs/dataset/object-detection.md

```diff
@@ -233,40 +233,11 @@ For details on what is recommended within your model results, see the relevant d
 
 ### Step 4: Upload Object Detection Model Results
 
-=== "Web App"
```

Comment:
Thanks for picking this up.

This is sort of gray area in that, in this example it actually can work because the files in `s3://kolena-public-examples/coco-2014-val/person-detection/results/` were actually exported from the Kolena web app after someone ran `upload_object_detection_results` -- i.e. this is a special case where the data in the csv files are indeed outputs from the sdk clients. 

I think there's merit in this example to keep the web app upload part so that an OD customer can set up a dataset fairly easily via the UI, but maybe add a note to say that generally only sdk client uploads would work for OD. Let me know what you think!

Reply:
It's a good point. Let me think about this

Reply:
I updated the docs based on your feedback. Could you take a look when you have a chance?